velox/connectors/hive/HiveConnector.h

```diff
@@ -270,12 +270,12 @@ class HiveConnectorFactory : public ConnectorFactory {
       "hive-hadoop2";
 
   HiveConnectorFactory() : ConnectorFactory(kHiveConnectorName) {
-    dwio::common::FileSink::registerFactory();
```

Comment:
I wonder why do we need to register local file sink in the Hive Connector? Shouldn't the application using Hive connector register whatever file system it is going to use?

Reply:
My best guess is it just tries to make sure at least one DataSink, the LocalFileSink, is registered, as Velox itself also needs it for, ex., unit tests.

Reply:
Will follow up with a separate PR and move it to the right place.